tests/SIL.LCModel.Tests/DomainServices/ParagraphParserTests.cs

```diff
@@ -687,6 +687,11 @@ public virtual IWfiAnalysis BreakIntoMorphs(int iSegment, int iSegForm, ArrayLis
 						// just set the Form.
 						wmb.Form.SetVernacularDefaultWritingSystem((string)morphForm);
 					}
+					else if (morphForm is ITsString)
+					{
+						var tssMf = (ITsString)morphForm;
+						wmb.Form.set_String(TsStringUtils.GetWsAtOffset(tssMf, 0), tssMf);
+					}
```

Comment:
https://gerrit.lsdev.sil.org/#/c/5799/10/Src/LexText/Interlinear/ITextDllTests/InterlinearExporterTests.cs@228 requires this.